deployment/aws/cdk/config.py

```diff
@@ -19,14 +19,18 @@ class StackSettings(pydantic.BaseSettings):
     # or https://developmentseed.org/titiler/advanced/performance_tuning/
     env: Dict = {
         "CPL_VSIL_CURL_ALLOWED_EXTENSIONS": ".tif,.TIF,.tiff",
+        "CPL_VSIL_CURL_CACHE_SIZE": "200000000",
```

Comment:
The tradeoff as a default is that it makes it faster for assets without sidecar files (because GDAL doesn't have to make the http requests to probe if they exist), but it makes performance a lot slower for files that do have sidecar files.